packages/cyberstorm/src/components/Layout/PackageDetailLayout/PackageManagementForm/PackageManagementForm.tsx

```diff
@@ -11,6 +11,7 @@ export function PackageManagementForm(props: PackageManagementFormProps) {
   return (
     <div className={styles.root}>
       <div className={styles.section}>
+        ADD ALERT HERE
```

Comment:
Eh?

Reply:
It's so that I remember to put it there tomorrow.

Reply:
https://github.com/thunderstore-io/thunderstore-ui/pull/709 Alerts aren't merged yet

Reply:
I'll remove it, maybe I can remember it

Reply:
At least make it a TODO prefixed comment